package.json

```diff
@@ -52,6 +52,7 @@
     "eslint-plugin-react": "^7.10.0",
     "eslint-plugin-standard": "^3.1.0",
     "gh-pages": "^1.2.0",
+    "hoist-non-react-statics": "^3.3.0",
```

Comment:
Why `devDependencies`? This would be part of the release code of Apps and should be a regular dependency, correct?